src/components/DetailsPanel/index.js

```diff
@@ -1,36 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import {
-  StyleSheet, View, ActionSheetIOS, Text,
+  StyleSheet, View, Text,
 } from 'react-native';
 import { ScrollView, TouchableOpacity } from 'react-native-gesture-handler';
 import {
   Header, Right, Title, Left,
 } from 'native-base';
-import { Entypo, SimpleLineIcons } from '@expo/vector-icons'; // eslint-disable-line import/no-extraneous-dependencies
+import { SimpleLineIcons } from '@expo/vector-icons'; // eslint-disable-line import/no-extraneous-dependencies
 import { shape } from 'prop-types';
 
 import Colors from '../../constants/Colors';
+import SortingHeader from './SortingHeader';
 
 const DetailsPanel = ({ navigation, collection }) => {
-  const handlePressSortIcon = () => {
-    ActionSheetIOS.showActionSheetWithOptions(
-      {
-        options: ['Cancel', 'Sort Records By Category', 'Sort Records By Date Saved'],
-        cancelButtonIndex: 0,
-        userInterfaceStyle: 'dark',
-      },
-      (buttonIndex) => {
-        if (buttonIndex === 0) {
-          // cancel action
-        } else if (buttonIndex === 1) {
-          // Sort Records By Category;
-        } else if (buttonIndex === 2) {
-          // Sort Records By Date Saved;
-        }
-      },
-    );
+  const defaultSortingState = {
+    'record-type': {
+      isPicked: true,
```

Comment:
if onnly one thing can be picked at a time, then sort `sort_field` should possibly be a single value.  (which may mean that types map to SORT_DIR, and not Objects)

this would especially be convenient if this eventually has to be in Redux.  (to store w/ each collection)